### PR TITLE
Use JS string.startsWith. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: src/modules.js [hunk 1]
```diff
@@ -284,7 +284,7 @@ function cDefine(key) {
 }
 
 function isFSPrefixed(name) {
-  return name.length > 3 && name[0] === 'F' && name[1] === 'S' && name[2] === '_';
+  return name.length > 3 && name.startsWith('FS_');
```
Comment (kripken) [inline src/modules.js:287]:
```suggestion
  return name.startsWith('FS_');
```

**Review by kripken — COMMENTED** [review]
lgtm with comment

File: src/modules.js (hunk 1 above)
Comment (kripken) [reply src/modules.js:287]: (I don't think we need to ignore `FS_` itself here.)

Comment (sbc100) [reply src/modules.js]: In that case I can just remove the function completely and inline it.

**Pull request merged**